lib/bundle/brewfile.rb

```diff
@@ -2,22 +2,22 @@
 
 module Bundle
   module Brewfile
+    ENV_KEY = "HOMEBREW_BUNDLE_FILE"
```

Comment:
Don't think this is worth making a constant for a single use in this file.

Reply:
For one I agree. There are two so maybe its arguable.